init.sql

```diff
@@ -0,0 +1,26 @@
+CREATE ROLE librarian WITH LOGIN SUPERUSER PASSWORD 'librarian';
+ALTER DATABASE library OWNER TO librarian;
+
+CREATE TABLE person (
+    id          INT         PRIMARY KEY,
```

Comment:
For primary key better constructor is `id serial Primary Key` it automatically create a sequence, and don't need manually generate in the insert process.